diffpy/pdfmorph/tests/test_morphsequence.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
usually we use the pattern that the test module that that tests modeul `<filename>` is called `test_<filename>`.  This pattern may not be being used in pdfmorph which whose provenance is older, but when we edit code, we like the edits to follow the new standards if possible.  I am not sure if it makes sense here, but if it does can we use this new pattern?

Reply:
I'll move these to a file called test_pdfmorphapp